src/code/z_eff_blure.c

```diff
@@ -705,6 +705,7 @@ void EffectBlure_DrawSmooth(EffectBlure* this2, GraphicsContext* gfxCtx) {
         }
     }
 
+close_disps:
```

Comment:
Based on OoT debug rom and also MM debug rom, these actually were early returns since this Graph_CloseDisps function doesn't get called in those control flows. This only works since this is retail and doesn't use that function.

Reply:
Ah, this one I wasn't sure about. I should have checked OoT decomp for that 😅 
I can revert that change if that makes more sense.

Would this be a candidate for `// @bug Skips CLOSE_DISPS` like the other places, or just leave as is?

Reply:
I would say add bug tags to both

Reply:
Updated